app/cdap/components/Administration/TetheringTabContent/index.tsx

```diff
@@ -41,7 +78,29 @@ const AdminTetheringTabContent = () => {
         })}
       />
       <AdminTetheringTabContainer>
-        {Theme.odf === true ? <OdfTetheringConnections /> : <CdfTetheringConnections />}
+        {Theme.odf === true ? (
```

Comment:
`Theme.odf` or `!!Theme.odf` if you want to make sure

Reply:
We had an annoying custom rule defined in tsconfig at my previous job that avoided inferring truthy/falsy values from variables. I guess I still subconsciously have that in mind.